src/electronApp/angularApp/components/connect-form/connect-form.component.html

```diff
@@ -2,21 +2,33 @@
   <h1>Hello!</h1>
 
   <h2 class="instructions">Enter printer's IP address to continue</h2>
-  <span *ngIf="isError" onkeypress="OnkeyPress" class="error warningColor">Failed to connect to printer, is the address correct?</span>
+  <span *ngIf="isError"
+        onkeypress="OnkeyPress"
```

Comment:
Not your change, it was already here, but I don't think this click handler actually does anything

Reply:
Yeah doesn't look like it, I removed it 18dafab